### PR TITLE
PR to the official repository for Ergo

### PR DESCRIPTION
**Pull request by zargarzadehm**

### CODE REVIEW

**Comment by sham789** [comment]
Hi! I don't see that private keys initialization is handled. Your keys init function should be provided here - [link](https://github.com/Gravity-Tech/gravity-core/blob/a938bdb2851c29894491723785e1a22c5c555d7a/config/config.go#L52). 

Moreover, please double check that your adapter features are provided in all necessary places by searching exising adapter keyword. (I would rely on "Ethereum" for i.g.)

**Comment by sham789** [comment]
Looks good. I think we need a couple of more tests for:
1. Consuls signature collection & submission for Gravity contract.
2. Oracles signature collection & submission for Nebula contract.
3. Data extraction flow (test covers oracle node behaviour).

File: cmd/gravity/commands/ledger.go [hunk 1]
```diff
@@ -87,6 +87,11 @@ var (
 				ChainType:              "waves",
 				GravityContractAddress: "3PLpMu2cAg618e7xXYHtckFJjFZksPFHoLm",
 			},
+			account.Ergo.String(): {
+				NodeUrl: "http://10.10.10.4:9016",
```
Comment (sham789) [inline cmd/gravity/commands/ledger.go:91]: 10.* seems like a internal IP

Comment (zargarzadehm) [reply cmd/gravity/commands/ledger.go:91]: It's right
I will close this PR and after the final update and final test, I will send the PR again.

**Pull request closed without merging**